compiler/q-implant/src/QImplant.cpp

```diff
@@ -32,6 +32,10 @@ using namespace q_implant;
 
 namespace
 {
+// TODO: Find opcodes which can be appended in conversion
+std::set<luci::CircleOpcode> appendable_operator_opcode{
```

Comment:
plz put this NONE STATIC function or struct/class member.

Reply:
Replaced it to newly created class member `forward_qparam`